llama-index-core/llama_index/core/llms/function_calling.py

```diff
@@ -27,6 +22,10 @@ class FunctionCallingLLM(LLM):
 
     """
 
+    def __init__(*args, **kwargs) -> None:
+        # Help static checkers understand this class hierarchy
+        super().__init__(*args, **kwargs)
```

Comment:
Huh, TIL

Reply:
Yeah Pyright gets lost without an explicit __init__ when a pydantic model is part of the hierarchy